tests/test-pad-reflect-1d.cpp

```diff
@@ -85,10 +87,10 @@ void test_pad_reflect_1d(bool use_gpu) {
 
     // Test cases for different padding configurations
     {
-        params = (struct ggml_init_params){
-            .mem_size   = 16*1024*1024,
-            .mem_buffer = NULL,
-            .no_alloc   = true,
+        params = ggml_init_params{
+            16*1024*1024,
+            nullptr,
+            true
```

Comment:
The pattern that we use is to put the members in comments like this:

```suggestion
        params = ggml_init_params{
            /*.mem_size   =*/ 16*1024*1024,
            /*.mem_buffer =*/ nullptr,
            /*.no_alloc.  =*/ true
```

Reply:
> The pattern that we use is to put the members in comments like this:

Thanks, I applied this to all the instances.

Unfortunately, there's a risk that these kind of comments and the structure will go out-of-sync. Enabling C++20 and using the designated initializers would provide more safety here, and since the set of compile which support C++17 but not C++20 is getting really small, hopefully the upgrade to C++20 will be reconsidered in the future.